### PR TITLE
fix: tabs in RTL mode

### LINKED ISSUE #7245: [Bug Report] Tabs center not working correctly in RTL
### Versions and Environment
**Vuetify:** 1.5.14
**Vue:** 2.6.10
**Browsers:** Chrome 74.0.3729.157
**OS:** Linux x86_64

### Steps to reproduce
Use tabs component and add a center prop in RTL mode

### Expected Behavior
Should work like LTR in center

### Actual Behavior
_

### Reproduction Link
<a href="https://codepen.io/miladmeidanshahi/pen/xNLBqZ?&editable=true&editors=101" target="_blank">https://codepen.io/miladmeidanshahi/pen/xNLBqZ?&editable=true&editors=101</a>

### PR DESCRIPTION
**Pull request by jacekkarczmarczyk**
## Motivation and Context
fixes #7245 and other issues:

- https://codepen.io/anon/pen/pmLGao
- tabs are initially not visible
- click right arrow as long as tabs will be aligned to the right
- click left arrow (tabs shift correctly), click left arrow again (tabs go back to the initial position)
- click left arrow, click the most left tab, tabs get crazy
- click anywhere, nothing works as it should

## How Has This Been Tested?
visually, unit

## Markup:

<details>

```vue
<template>
  <v-app>
    <v-content>
      <v-tabs show-arrows>
        <v-tab v-for="i in 50" :key=i>{{ i }}</v-tab>
      </v-tabs>
      <v-tabs centered>
        <v-tab v-for="i in 4" :key=i>{{ i }}</v-tab>
      </v-tabs>
    </v-content>
  </v-app>
</template>

<script>
export default {
  created () {
    this.$vuetify.rtl = true
  }
}
</script>
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VSlideGroup/VSlideGroup.ts [hunk 1]
```diff
@@ -274,37 +266,52 @@ export const BaseSlideGroup = mixins<options &
     overflowCheck (e: TouchEvent, fn: (e: TouchEvent) => void) {
       this.isOverflowing && fn(e)
     },
-    scrollIntoView () {
+    scrollIntoView /* istanbul ignore next */ () {
```
Comment (sh7dm) [inline packages/vuetify/src/components/VSlideGroup/VSlideGroup.ts]: Why ignoring here?

File: packages/vuetify/src/components/VSlideGroup/VSlideGroup.ts [hunk 2]
```diff
@@ -315,12 +322,13 @@ export const BaseSlideGroup = mixins<options &
...
-    setOverflow () {
-      this.isOverflowing = this.widths.wrapper < this.widths.content
+      this.scrollOffset = this.calculateNewOffset(location, {
+        // Force reflow
+        content: this.$refs.content ? this.$refs.content.clientWidth : 0,
+        wrapper: this.$refs.wrapper ? this.$refs.wrapper.clientWidth : 0
+      }, this.$vuetify.rtl, this.scrollOffset)
     },
-    setWidths () {
+    setWidths /* istanbul ignore next */  () {
```
Comment (sh7dm) [inline packages/vuetify/src/components/VSlideGroup/VSlideGroup.ts]: Why ignoring?

File: packages/vuetify/src/components/VTabs/VTabs.sass [hunk 3]
```diff
@@ -90,6 +90,18 @@
...
+      margin-left: auto
+
+    .v-tabs-bar__content > *:first-child:not(.v-tabs-slider-wrapper)
+      margin-left: 0
+      margin-right: auto
+
+    .v-tabs-slider-wrapper + *
+      margin-left: 0
+      margin-right: auto
 .v-tabs--fixed-tabs
```
Comment (sh7dm) [inline packages/vuetify/src/components/VTabs/VTabs.sass]: Space after block please
```suggestion
 .v-tabs--fixed-tabs

```

File: packages/vuetify/src/components/VSlideGroup/VSlideGroup.ts (hunk 2 above)
Comment (jacekkarczmarczyk) [reply packages/vuetify/src/components/VSlideGroup/VSlideGroup.ts]: Same reason as why `scrollTo` and `newOffset` were previously ignored. Not sure what was the actual reason though :P

**Pull request merged**